libs/designsystem/src/lib/components/chart/chart-helper.ts

```diff
@@ -1,17 +1,18 @@
-import { Options, chart } from 'highcharts';
+import { ElementRef, Injectable } from '@angular/core';
+import { chart, Options } from 'highcharts';
 import * as Highcharts from 'highcharts';
-import { ElementRef } from '@angular/core';
+import * as accessibility from 'highcharts/modules/accessibility';
+import * as exportData from 'highcharts/modules/export-data';
+import * as exporting from 'highcharts/modules/exporting';
 
 // Docs on importing accessibility: https://www.highcharts.com/docs/chart-concepts/accessibility
-import * as exporting from 'highcharts/modules/exporting';
-import * as exportData from 'highcharts/modules/export-data';
-import * as accessibility from 'highcharts/modules/accessibility';
 
 declare var require: any;
 require('highcharts/highcharts-more')(Highcharts);
 require('highcharts/modules/solid-gauge')(Highcharts);
 require('highcharts/modules/broken-axis')(Highcharts);
 
+@Injectable()
```

Comment:
Why this? 🤔

Reply:
I don't know, but the Angular migration tool added it.

https://update.angular.io/?l=2&v=9.1-10.2

> As of Angular 9, enforcement of @Injectable decorators for DI is stricter and incomplete provider definitions behave differently. *Read more. ng update will migrate you automatically.

*Read more: https://v9.angular.io/guide/migration-injectable

Reply:
Ok, that's sounds fair 👍🏻 
TL;DR: It's basically saying: "You SHOULD ALWAYS decorate services with `@Injectable()` and we plan to make it a requirement in the future, so we'll just help you out and future proof your code base"